features/elasticsearch/graphql.feature

```diff
@@ -0,0 +1,100 @@
+Feature: GraphQL query support
+
+  @elasticsearch
+  Scenario: Execute a GraphQL query on an ElasticSearch model with SubResources
+    When I send the following GraphQL request:
+    """
+    query {
+      users {
+        edges {
+          node {
+            id,
+            gender,
+            tweets {
+              edges {
+                node {
+                  id,
+                  message,
+                }
+              }
+            }
+          }
+        }
+      }
+    }
```

Comment:
why is the query so complex? Couldn't you use colleciton instead of edges + node ?

```
query {
  esProducts {
    collection {
      id, attributes { collection { id }}
    }
  }
}
```

Reply:
The `collection` doesn't seem available on the User existing model. I prefered using an existing models instead of creating new ones

Reply:
but collection is needed to go through the new DataProvider I think, does the test fail without the new provider?

Reply:
I ran this test on 2.6 branch and the test was failing. It runs successfully on your branch